pkg/storaged/storage-controls.jsx

```diff
@@ -157,7 +157,7 @@ export class StorageOnOff extends React.Component {
     render() {
         const self = this;
 
-        function onChange(val) {
+        function onChange(_event, val) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230519-121108-54b4f0d5-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)